crates/swc/tests/tsc-references/objectRest2_es5.1.normal.js

```diff
@@ -58,24 +58,29 @@ function _objectSpread(target) {
 }
 function rootConnection(name) {
     return {
-        resolve: _asyncToGenerator(regeneratorRuntime.mark(function _callee(context, args) {
-            var objects;
-            return regeneratorRuntime.wrap(function _callee$(_ctx) {
-                while(1)switch(_ctx.prev = _ctx.next){
-                    case 0:
-                        _ctx.next = 2;
-                        return {
-                            objects: 12
-                        };
-                    case 2:
-                        objects = _ctx.sent.objects;
-                        return _ctx.abrupt("return", _objectSpread({}, connectionFromArray(objects, args)));
-                    case 4:
-                    case "end":
-                        return _ctx.stop();
-                }
-            }, _callee);
-        }))
+        resolve: (function() {
```

Comment:
This seems like a regression of IIFE optimization.

Reply:
Hi @kdy1 

https://github.com/swc-project/swc/blob/56d1f03556f6145ef2e454fac3a4450b06034354/crates/swc_ecma_parser/tests/typescript/tsc/objectRest2/input.ts#L5-L15


We should make sure:
```diff
+ expect(rootConnection('test').resolve.length).toBe(2);
```

Reply:
Oh, I see. Thank you!
Then this is a field of the minifier, not transform.